packages/expo-camera/ios/CameraViewModule.swift

```diff
@@ -119,7 +119,9 @@ public final class CameraViewModule: Module, ScannerResultHandler {
       }
 
       Prop("mute") { (view, muted: Bool?) in
-        view.isMuted = muted ?? false
+        if let muted, view.isMuted != muted {
```

Comment:
I believe it is handled by module core (or RN) internally -- removing it will call that prop and configures accordingly. Removing it is basically just setting it back to `false`.

Reply:
Actually, it's taken care of by a function the parses the props so this is fine. Just note that removing a prop will pass null as it's value.

Reply:
I see.. yeah it should be fine since that only happens in dev fast refresh. We still need to condition somehow to not call the config if it didn't change since we need to call the same code in the `setCameraMode` block

Reply:
Actually, it does correctly pass `Bool` if omitted. Looks like it's handled by default by the module i.e. `null` is never passed here.
<img width="217" alt="Screenshot 2024-05-10 at 15 38 49" src="https://github.com/expo/expo/assets/6686328/3049391b-7c21-4e46-b1c8-497ac81363fd">